src/locales/intl.js

```diff
@@ -0,0 +1,125 @@
+import 'intl'
+
+const EN_US_LOCALE = {
+  'localeIdentifier': 'en_US',
+  'decimalSeparator': '.',
+  'quotationBeginDelimiterKey': '“',
+  'quotationEndDelimiterKey': '”',
+  'currencySymbol': '$',
+  'currencyCode': 'USD',
+  'groupingSeparator': ','
+}
+let locale = EN_US_LOCALE
+
+const intlHandler = {
+  /**
+   * Formats number input according to user locale
+   * Allows decimalSeparator at the end of string
+   * @param number  - Native style (JS) number
```

Comment:
Looks like this can take a number or a string. Be careful as some numbers a big ints and would overflow a JS number.